pkg/policy/l4.go

```diff
@@ -813,6 +813,7 @@ func createL4Filter(policyCtx PolicyContext, peerEndpoints api.EndpointSelectorS
 	}
 
 	// already validated via L4Proto.Validate(), never "ANY"
+	// NOTE: "ANY" for wildcarded port/proto!
 	u8p, _ := u8proto.ParseProtocol(string(protocol))
```

Comment:
Not quite yet, as L4Filter still converts ANY to the supported protocols. Unless a later commit in this PR changes this we should change that for fully wildcarded port/proto as it will save on space in the map.

Reply:
This would be safe to do, if the datapath will limit the proto to known supported one. Otherwise, in theory, there could be a `proto` which does not have ports in the traditional sense and the datapath could misbehave, if there is a ANY proto with port 80, etc.

Reply:
Let's address in a different PR.